### PR TITLE
[ADD] Essentials: Keyboard shortcuts

### PR DESCRIPTION
**Pull request by jero-odoo**
Docs task: (https://www.odoo.com/mail/view?model=project.task&res_id=4140097&access_token=3910e656-b79f-415c-a7eb-24d390e2606e)

Adding new documentation with an overview of the most common/useful keyboard shortcuts in Odoo.

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hey @kekiss this is ready for peer review. Thanks!

File: content/applications/essentials/keyboard_shortcuts.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`F`
+   * - Share a Knowledge article
+     - :kbd:`Alt` + :kbd:`Shift` + :kbd:`S`
+     - :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`S`
+   * - Open command palette
+     - :kbd:`Ctrl` + :kbd:`K`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`K`
+
+.. tip::
+  After opening the command palette, search use the following |kbs|:
```
Comment (kekiss) [inline content/applications/essentials/keyboard_shortcuts.rst:80]:
```suggestion
  After opening the command palette, search using the following |kbs|:
```

**Review by kekiss — APPROVED** [review]
Fantastic job on this @jero-odoo ! :star2: I left two small comments/suggestions, but aside from those little things, this looks great!

**Comment by jero-odoo** [comment]
Thanks Kara!

@ksc-odoo this is ready for final review. Can you make sure to test the command for changing views? (Line 49). Kara was having issues with it, but it is working on my computer ok. 
Thank you!

File: content/applications/essentials/keyboard_shortcuts.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+.. |Kbs| replace:: Keyboard shortcuts
+.. |os| replace:: operating system
+
+Users in Odoo can utilize several |kbs| to navigate through modules, execute actions, and manage
+data.
+
+.. image:: keyboard_shortcuts/menu-shortcuts.png
+   :align: center
+   :alt: A selection of keyboard shortcuts in Odoo.
+
```
Comment (ksc-odoo) [inline content/applications/essentials/keyboard_shortcuts.rst:15]:
```suggestion
```
one too many blank lines

File: content/applications/essentials/keyboard_shortcuts.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`F`
+   * - Share a Knowledge article
+     - :kbd:`Alt` + :kbd:`Shift` + :kbd:`S`
+     - :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`S`
+   * - Open command palette
+     - :kbd:`Ctrl` + :kbd:`K`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`K`
+
+.. tip::
+  After opening the command palette, search using the following |kbs|:
```
Comment (ksc-odoo) [inline content/applications/essentials/keyboard_shortcuts.rst:80]:
```suggestion
.. tip::
   After opening the command palette, search using the following |kbs|:
```

File: content/applications/essentials/keyboard_shortcuts.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+  - :kbd:`/`: search for menus, applications, and modules.
+  - :kbd:`@`: search for users.
+  - :kbd:`#`: search for **Discuss** channels.
+  - :kbd:`?`: search for **Knowledge** articles.
+
+  Enter a name or term in the search bar, or use the arrow keys to scroll through the available
```
Comment (ksc-odoo) [inline content/applications/essentials/keyboard_shortcuts.rst:87]:
```suggestion
  Enter a name (or term) in the search bar, or use the arrow keys to scroll through the available
```
optional: added parenthetical here to provide a cleaner pacing through the sentence

File: content/applications/essentials/keyboard_shortcuts.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  options. Then click :kbd:`Ctrl` + :kbd:`Enter` to open the selected app, module, or menu in a new
```
Comment (ksc-odoo) [inline content/applications/essentials/keyboard_shortcuts.rst:88]:
```suggestion
  options. Then, click :kbd:`Ctrl` + :kbd:`Enter` to open the selected app, module, or menu in a new
```

**Review by ksc-odoo — APPROVED** [review]
hey @jero-odoo -- just finished my Final Review of this super straightforward PR. Nicely done. Just have a few, super-minor comments that require your attention. But, once you address those, you can feel free to tag this for Tech Review. Thanks! 👍

**Comment by jero-odoo** [comment]
Thanks KC!
@samueljlieber this is ready for tech review. Thanks!

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo! Nice work on this new essentials doc, I think this will be a great reference to have :)

I know I have a lot of changes, so please bear with me 😄
- Please remove the shorthands for "kbs", "Kbs" and "os", as these are kind of unnecessarily complex since this doc is under 100 lines
- Make version specific edits to include the shortcuts that were introduced in 17.2 and 17.4 (can do this on the FWP PRs, LMK if you want help)
- (Optional) Consider indicating which view these shortcuts are available on (change view shortcut is only available when multiple views are avail). This is kinda covered by the important admonition

I tested each shortcut on macOS, and corrected them in my suggestions, so please feel free to merge once you address my suggestions!

Thank you for your work!
.....
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**